tests/robot-tests/tests/libs/common.robot

```diff
@@ -25,7 +25,7 @@ do this on failure
         prompt to continue
     END
     IF    ${prompt_to_continue_on_failure} == 0
-        set selenium timeout    3
```

Comment:
You could just remove this IF END block completely, since you're not changing the implicit wait time.